lib/src/main/java/com/otaliastudios/transcoder/source/DefaultDataSource.java

```diff
@@ -189,13 +195,13 @@ public void releaseTrack(@NonNull TrackType type) {
     public long seekTo(long desiredPositionUs) {
         boolean hasVideo = mSelectedTracks.contains(TrackType.VIDEO);
         boolean hasAudio = mSelectedTracks.contains(TrackType.AUDIO);
-        LOG.i("seekTo(): seeking to " + (mOriginUs + desiredPositionUs)
-                + " originUs=" + mOriginUs
-                + " extractorUs=" + mExtractor.getSampleTime()
-                + " externalUs=" + desiredPositionUs
-                + " hasVideo=" + hasVideo
-                + " hasAudio=" + hasAudio);
         if (hasVideo && hasAudio) {
+            LOG.i("seekTo(): seeking to " + (mOriginUs + desiredPositionUs)
```

Comment:
let's cover everything with VERBOSE check for now.
We'll figure out our logger later.

Reason: The string gets created and computed unconditionally and unnecessarily even when the logger isn't supposed to log it. Hence it'll affect performance in production.

Reply:
got it